centreon/www/class/centreonGraph.class.php

```diff
@@ -272,13 +272,20 @@ public function __construct($user_id, $index = null, $debug = 0, $compress = nul
             }
             $DBRESULT->closeCursor();
 
-            foreach ($odsm as $mid => $val) {
-                if (!isset($metrics_cache[$mid])) {
-                    $DBRESULT = $this->DB->query(
-                        "INSERT INTO `ods_view_details`
-                            (`metric_id`, `contact_id`, `all_user`, `index_id`)
-                            VALUES ('" . $mid . "', '" . $this->user_id . "', '0', '" . $this->index . "');"
-                    );
+            $insertViewDetailsStatement = $this->DB->prepareQuery(
```

Comment:
Catch CentreonDbException and log the error with CentreonLog with business informations and a context please

Reply:
this PR is a backport (as the other ones) and prepareQuery will already logs information about the errors. But I can create a ticket to handle this change also on develop (already merged). As this tickets is kinda urgent

Reply:
It's not the same log, here it's to have a business log. in centreondb it's a technical log ;) 
Ok I understand, can you be more careful next time please ? Thanks.